src/hardware/adc.rs

```diff
@@ -133,55 +139,34 @@ type Adcs = ad7172::Ad7172<Spi<SPI4, Enabled>>;
 /// * `sync` - ADC sync pin (shared for all adc phys).
 pub struct AdcPins {
     pub spi: (
-        PE2<Alternate<AF5>>,
-        PE5<Alternate<AF5>>,
-        PE6<Alternate<AF5>>,
+        gpioe::PE2<gpio::Alternate<gpio::AF5>>,
```

Comment:
To avoid `use *`, generally reducing the size of the namespace, and add context. This makes it clear that `Alternate` is actually the gpio alternate function, not something else that may be around. C.f. the `Enabled` (both from `spi` and `timer` IIRC) problem a while ago.